db/migrate/20170504145330_sync_whitehall_public_updated_at.rb

```diff
@@ -0,0 +1,29 @@
+class SyncWhitehallPublicUpdatedAt < ActiveRecord::Migration[5.0]
+  def change
+    schema_names = %w(gone redirect special_route speech statistical_data_set take_part topical_event_about_page)
```

Comment:
The complete number of Whitehall records where the `first_published_at` is more recent that the `public_updated_at` is ~600,000, I'm addressing the simple cases first here on a per-format basis.
Formats like `gone` and `redirect` are fairly low risk and the others I've included here either don't render these dates or have a correct `first_published_at` value which can be used as the `public_updated_at`. Basically this should cover the quick wins.